FreeITCourse/src/lesson05/MetodTask21.java

```diff
@@ -0,0 +1,49 @@
+package lesson05;
+
+
+import java.util.Date;
+
+
+// Задание 21
+// Напишите три цикла выполняющих многократное сложение строк,
+// один с помощью оператора сложения и String,
+// другой с помощью StringBuilder и метода append,
+// а также аналогино для StringBuffer.
+// Сравните скорость их выполнения.
+public class MetodTask21 {
+    public static void main(String[] args) {
```

Comment:
please use
System.nanoTime()
for getting more correct time intervals, it should make comparison easier.
run application at least 3 times and give some description what is more efficient

Reply:
corrected